### ISSUE TITLE
Allow to add custom registers to liquid

### ISSUE THREAD

**Issue by ulitiy** [locomotivecms/engine#1028]
We can override `locomotive_default_registers`, but it can be better if we have a config option.

**Comment by did** [comment]
@ulitiy you're absolutely right. This has to be implemented in [Steam](https://github.com/locomotivecms/steam/issues/40).
